README.md

```diff
@@ -468,6 +468,9 @@ fastify.get('/responseDescription', {
   }
 }, () => {})
 ```
+
+Additionally, if you provide a `$ref` in your response schema but no description, the reference's description will be used as a fallback.
```

Comment:
Please state the `$ref` currently only support the format of matching `$id`.
It cannot lookup the fragment part.